addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -662,8 +662,13 @@ export function makeDraggableHook(hookParams) {
                 // https://bugzilla.mozilla.org/show_bug.cgi?id=1352061
                 // https://bugzilla.mozilla.org/show_bug.cgi?id=339293
                 safePrevent(ev);
-                if (document.activeElement && !document.activeElement.contains(ev.target)) {
-                    document.activeElement.blur();
+                const eventDocument = ev.target.ownerDocument;
```

Comment:
There is a possibility that ownerDocument returns `null` if the element is the document itself.
This shouldn't happen but we could add a small defensive measure as a safeguard.

Reply:
Ah indeed, made it use the target itself as fallback granted the target has "activeElement"